virtcontainers/sandbox.go

```diff
@@ -39,6 +40,10 @@ const (
 	// vmStartTimeout represents the time in seconds a sandbox can wait before
 	// to consider the VM starting operation failed.
 	vmStartTimeout = 10
+	// waitRetry represents retry times wait for vm exit
+	waitRetry = 50
+	// waitVMExit represents times wait for vm exit in one loop
+	waitVMExitTime = 200
```

Comment:
since this is time, please convert it to milliseconds `200 * time.Millisecond` or mention that this value is in milliseconds